runtime/runtime.go

```diff
@@ -1305,17 +1322,24 @@ func (r *interpreterRuntime) newLogFunction(runtimeInterface Interface) interpre
 	return func(invocation interpreter.Invocation) trampoline.Trampoline {
 		message := fmt.Sprint(invocation.Arguments[0])
 		wrapPanic(func() {
-			runtimeInterface.Log(message)
+			err := runtimeInterface.Log(message)
+			if err != nil {
+				panic(err)
+			}
 		})
```

Comment:
The `wrapPanic` should be only around the interface call:

```go
var err error
wrapPanic(func() {
	err = runtimeInterface.Log(message)	
})
if err != nil {
	panic(err)
}
```

Reply:
applied at 
19d823e